sycl/plugins/level_zero/usm_allocator.cpp

```diff
@@ -617,6 +617,10 @@ void *USMAllocContext::USMAllocImpl::allocate(size_t Size, size_t Alignment) {
   if (Alignment <= 1)
     return allocate(Size);
 
+  // L0 does not support alignments > 64KB, so we don't either.
+  if (Alignment > 65536)
+    Alignment = 65536;
```

Comment:
This will silently NOT satisfy the requested alignment. Should we give an error instead?

Reply:
I was also debating with myself about that.
What type of error would that be? There is no existing PI_ERROR_ that could be used, other than UNKNOWN.

Reply:
We could just use PI_INVALID_VALUE or even create a new PI error code (later will need some design thought how to not overlap with existing errors still based of OpenCL errors).